tests/test_routes.py

```diff
@@ -85,6 +91,24 @@ def is_path_and_method_in_list(path, method):
             is_path_and_method_in_list(path="/payment-method/:id", method="PUT")
         )
 
+
+  
+    def test_update_payment(self):
+        """It should Update an existing Payment Method"""
+        # create a payment to update
+        test_payment = PaymentMethodFactory()
```

Comment:
I doubt that this will work. we can't add just an instance of `PaymentMethod` because it's a base class for different payment methods. for the best reliability you should probably do something like:
```python
import random

...

test_payment_method_class = random.choice([CreditCardFactory, PayPalFactory])
test_payment_method = test_payment_method_class()
```